app/assets/javascripts/application.js.erb

```diff
@@ -798,6 +798,12 @@ $(document).on('click', '#terms_check_thank_you', function() {
   }
 });
 
+$(document).on('keydown', 'input.thank_you_field', function() {
+  if(event.code === "Tab") {
```

Comment:
- As per this new code, tab feature is completely disabled for this page.
- Tab should not be disabled and user should also be able to navigate through the page using this feature.